android/src/com/mapswithme/maps/search/SearchAdapter.java

```diff
@@ -176,6 +176,11 @@ else if (!TextUtils.isEmpty(result.description.roadShields))
         }
       }
 
+      if (result.isHotel && result.stars != 0)
+      {
+        tail.append(" • ").append("★★★★★".substring(0, result.stars));
```

Comment:
We can have up to 7 stars (such hotels exist). Can we make safe code here, something like .fromNchars('★', result.stars) ?

Reply:
In that case we should also update iOS code, because there also 5 stars is the maximum result: [link](https://github.com/organicmaps/organicmaps/blob/eca232021ce5ac52fa59e3e4a9085bbc11ffe62d/iphone/Maps/UI/Search/TableView/MWMSearchCommonCell.mm#L45)

Reply:
Yes, that's actually how I discovered it, because iOS crashes now on this rare cases.